src/controllers/ioSignController.ts

```diff
@@ -126,6 +135,93 @@ export default class IoSignController {
       )()
     );
 
+  /**
+   * Create a Signature from a Signature Request
+   */
+  public readonly createSignature = (
+    req: express.Request
+  ): Promise<
+    | IResponseErrorInternal
+    | IResponseErrorValidation
+    | IResponseErrorNotFound
+    | IResponseSuccessJson<SignatureDetailView>
+  > =>
+    withUserFromRequest(req, async user =>
+      pipe(
+        req.body,
+        CreateSignatureBody.decode,
+        E.mapLeft(responseErrorValidation),
+        TE.fromEither,
+        TE.chainW(body =>
+          pipe(
+            sequenceS(TE.ApplySeq)({
+              signerId: pipe(
+                retrieveSignerId(this.ioSignService, user.fiscal_code),
+                TE.mapLeft(
+                  responseErrorInternal(
+                    "Error retrieving the signer id for this users"
+                  )
+                )
+              ),
+              userProfile: pipe(
+                profileWithEmailValidatedOrError(this.profileService, user),
+                TE.mapLeft(
+                  responseErrorInternal(
+                    "Error retrieving a user profile with validated email address"
+                  )
+                )
+              )
+            }),
```

Comment:
It's pretty similar to `createFilledDocument` one, maybe we can extract it into a common method?
Also, we can reuse `toErrorRetrievingTheSignerId`